src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int SIZE_OF_THE_ARRAY = 10;
+
+    public static void main(String[] args) {
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[SIZE_OF_THE_ARRAY];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < SIZE_OF_THE_ARRAY / 2) {
+                figures[i] = figureSupplier.getDefaultFigure();
+            } else {
+                figures[i] = figureSupplier.getRandomFigure();
```

Comment:
The distribution of the figures is uneven. Half of the figures are default, and the rest are random. To have an even distribution of figures, use a random boolean to determine whether to assign a default or random figure.

Reply:
The task requires that half of the array shoud contain the default figure